fastlane_core/lib/fastlane_core/ipa_file_analyser.rb

```diff
@@ -73,13 +73,20 @@ def self.fetch_info_plist_with_rubyzip(path)
     end
 
     def self.fetch_info_plist_with_unzip(path)
-      list, error, = Open3.capture3("unzip", "-Z", "-1", path)
+      entry, error, = Open3.capture3("unzip", "-Z", "-1", path, "*/Payload/*.app/Info.plist")
```

Comment:
There is a slight problem with using matching with `unzip` 😔 Matching on `*.app/Info.plist` does not handle the case with multiple apps in a `.app`. Like in the case of this WatchKit app 👇 

```
ContainsWatchApp/Payload/Sample.app/Watch/Sample WatchKit App.app/Info.plist                                                                    
ContainsWatchApp/Payload/Sample.app/Info.plist
```

I also tried to change it to `*/Payload/*.app/Info.plist` thinking that this might get the exact match of the main app Info.plist but `*.app` seems to behave like `**/*.app` 😬 

HOWEVER... I have a commit that think would fix that that I added 😊 If you would be able to review and test this for me that would be 🔥

Reply:
Hi Josh! Thank you for fixing this potential bug. I did a simple test on my side but the new code fails, will look into this later and come back to you!

Reply:
For some reason, `*/Payload/*.app/Info.plist` can't cover my case although theoretically it should also capture `Payload/xxx.app/Info.plist`.

Reply:
The latest code works for me now, so I assume we're clear for merge?